build.sbt

```diff
@@ -53,8 +51,8 @@ javaOptions in Universal ++= Seq(
 
 routesGenerator := InjectedRoutesGenerator
 
-scalacOptions := Seq("-unchecked", "-optimise", "-deprecation", "-target:jvm-1.8",
-    "-Xcheckinit", "-encoding", "utf8", "-feature", "-Yinline-warnings")
+scalacOptions := Seq("-unchecked", "-opt:l:method", "-deprecation", "-target:jvm-1.8",
```

Comment:
I'd remove `optimise` altogether unless there's a non-performance reason to have it?

Using this looks fine and the optimisation [is solid](https://www.scala-lang.org/news/2.12.0/#new-optimizer) but a few years ago the optimiser had been known to cause issues in libraries eg older versions of [Akka](https://doc.akka.io/docs/akka/2.4/intro/getting-started.html#Do_not_use_-optimize_Scala_compiler_flag)